src/server/migrations/20200608220954_actual_answers.js

```diff
@@ -0,0 +1,16 @@
+// TODO: This is a sample migration for demonstration. Remove before adding real ones.
+
+exports.up = function (knex) {
+  return knex.schema.createTable('actual_answers', (table) => {
+    table.increments('id');
+    table.integer('fk_game_user_id').unsigned();
+    table.foreign('fk_game_user_id').references('id').inTable('game_instances');
```

Comment:
Why did you wrote fk_game_user_id two times here?? One is just integer and another is foreign. If it's already foreign what is the logic to assign it as an integer here?

Reply:
because if you write it just as foreign it DB will not accept it, that its how fk works with knex, could you please reshow last week lesson  vedio... Zaki explain that ...

Reply:
that's good then.